lib/plugins/helper/toc.ts

```diff
@@ -27,7 +29,7 @@ function tocHelper(str: string, options: Options = {}) {
     list_number: true
   }, options);
 
-  const data = tocObj(str, { min_depth: options.min_depth, max_depth: options.max_depth });
+  const data = getAndTruncateTocObj(str, { min_depth: options.min_depth, max_depth: options.max_depth }, options.max_items);
```

Comment:
If the user does not need `max_items`, then there is no need to perform this operation everytime.

Reply:
fixed: c17db64

I thought it was necessary to avoid `tocObj()` appearing multiple times in different places, so I wrote the skip judgment at the beginning of `getAndTruncateTocObj()` instead of where you gave the example.